content/en-US/api/USBConfiguration/USBConfiguration.md

```diff
@@ -0,0 +1,29 @@
+---
+recipe: api-feature
+title: 'USBConfiguration'
+mdn_url: /en-US/docs/Web/API/USBConfiguration
+specifications: https://wicg.github.io/webusb/#configurations
+browser_compatibility: api.USBConfiguration
+---
+
+## Description
+
+The `USBConfiguration` interface of the WebUSB API provides information about a particular configuration of a USB device and the interfaces that it supports.
+
+## Constructor
+
+Creates a new `USBConfiguration` object which will be populated with information about the configuration on the provided `USBDevice` with the given configuration value.
+
+## Properties
+
+**`USBConfiguration.configurationValue`**
+
+Returns the configuration value of this configuration. This is equal to the `bConfigurationValue` field of the configuration descriptor provided by the device defining this configuration.
```

Comment:
Can you please add a link to what you think is the best page on `bConfigurationValue`?

Reply:
Unfortunately the USB-IF makes it difficult to link to sections of the official USB specifications, which would be the best reference here. Barring that the "USB in a NutShell" documentation from BeyondLogic has a good reference for many of these terms. What is the best way to reference this?

https://www.beyondlogic.org/usbnutshell/usb5.shtml#ConfigurationDescriptors

Reply:
Add it to the markdown like this:

```
[`bConfigurationValue `](https://www.beyondlogic.org/usbnutshell/usb5.shtml#ConfigurationDescriptors)
```

Reply:
Done.